### ISSUE TITLE
Using enrichments to fetch thumbnail URLs from record pages

### ISSUE THREAD

**Issue by revanar** [DigitalNZ/supplejack_manager#15]
Hi all,

I've been working with @telezoic on creating an instance of Supplejack in Docker for potential use with some IRs in Ontario and British Columbia.  We've been pretty active on the Supplejack Docker repo, but I figured I should start bringing questions that aren't specific to the Docker build to the appropriate place.

I was looking at ways to potentially fetch thumbnail URLs for OAI RIs that don't provide one with their metadata.  From looking at the documentation, it seems like this sort of job is what enrichment are built for, but I'm having trouble understanding from the documentation how I would go about performing this task.

Basically, I would want to create an enrichment that could visit the `source_url` of a page, discover the thumbnail URL in some fashion (perhaps by using JQuery to find `img[alt="Thumbnail"]`, or something similar, depending on the particular IR's DOM), and then save that URL as `thumbnail_url`.  Is this something that enrichment could do, and if so, how would you go about doing it?

Thank you for the help!

-Matt

**Comment by hapiben** [comment]
Hi @revanar 

I'm sorry it took long for me to get back to you. Is this related to the issue that you have raised in SJ docker repo?

Ben

**Comment by revanar** [comment]
Hi @hapiben,

Yeah, it is related to that one.  I opened this issue here because it didn't seem like my question was directly related to the Docker instance of Supplejack, but I wasn't sure if this was a place that you guys looked at frequently, since no other issues had ever been opened here, so I decided to open another more specific issue over on the Docker page, figuring you would see it.  @dan-dnz provided an answer there that seems like it will help a lot, though I'm still testing it currently!

I'll go ahead and close this issue there though.  Thanks for responding, @hapiben.
